import_flow_events.py

```diff
@@ -61,8 +61,8 @@
 """
 Q_CREATE_METADATA_TABLE = """
     CREATE TABLE IF NOT EXISTS flow_metadata (
-      flow_id VARCHAR(64) NOT NULL UNIQUE ENCODE lzo,
```

Comment:
> Do we lose anything by dropping the `UNIQUE` here?

I don't know for sure. Based on reading the docs beforehand I didn't think so but...

> Redshift doesn't enforce the uniqueness, but it does allegedly use it to speed up queries in some circumstances.

I didn't realise this. Where did you read about the performance part? I confess I am on shaky ground with much of the key-related stuff in redshift, some of the docs could with more concrete detail imho.